src/Sema.zig

```diff
@@ -364,6 +364,13 @@ pub const Block = struct {
 
     c_import_buf: ?*std.ArrayList(u8) = null,
 
+    /// If this is not `.none`, it refers to the innermost enclosing
+    /// If not `null`, this boolean is set when a `dbg_var_ptr` or `dbg_var_val`
+    /// instruction is emitted. It signals that the innermost lexically
+    /// enclosing `block`/`block_inline` should be translated into a real AIR
+    /// `block` in order for codegen to match lexical scoping for debug vars.
+    need_debug_scope: ?*bool = null,
```

Comment:
`?*bool` can be `.none` ?

Reply:
also does this need to be a pointer?

Reply:
Oops, typo, that's meant to say `null`. And yeah, it does, because a `Block` isn't necessarily a lexical block or AIR block instruction - it's an arbitrary WIP AIR body.